src/extension_constants.h

```diff
@@ -8,13 +8,16 @@
 /* No function definitions here, only potentially globally available defines as this is used by the
  * loader*/
 
-#define EXTENSION_NAME "timescaledb"
+#define EXTENSION_NAME "timescaledb"	  /* Name of the actual extension */
+#define EXTENSION_NAMESPACE "timescaledb" /* Namespace for extension objects */
 #define EXTENSION_FDW_NAME "timescaledb_fdw"
 #define TSL_LIBRARY_NAME "timescaledb-tsl"
 #define TS_LIBDIR "$libdir/"
 #define EXTENSION_SO TS_LIBDIR "" EXTENSION_NAME
 #define EXTENSION_TSL_SO TS_LIBDIR TSL_LIBRARY_NAME "-" TIMESCALEDB_VERSION_MOD
 
+#define EXTENSION_OPTION(NAME) (EXTENSION_NAMESPACE "." NAME)
```

Comment:
PostgreSQL often includes the term `make` in such macros (e.g., `MAKE_SQLSTATE`). Maybe `MAKE_EXTENSION_OPTION` might be more clear.

Reply:
It becomes very long though. Would `MAKE_EXTOPTION`  or `MAKE_TSLOPTION` be OK?

Reply:
+1 for `MAKE_EXTOPTION`

Reply:
Using `MAKE_EXTOPTION`.